.gitignore

```diff
@@ -55,13 +55,15 @@ util/Win_Check_Output.txt
 .vscode/last.sql
 .vscode/temp.sql
 .vscode/ipch/
+.vscode
```

Comment:
```suggestion
```

We shouldn't ignore this as it contains some recommended extensions and settings for working on QMK in VS Code.